lib/smartSurveyAPICall.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const axios = require('axios');
+const querystring = require('querystring');
+const SmartSurveyClient = require('smartsurvey-client')
```

Comment:
@sulthan-ahmed: Perhaps it's worth getting a linter in there. Are the HOF guys still using eslint?